aspnetcore/toc.yml

```diff
@@ -876,6 +876,6 @@
     - name: Logging (not ASP.NET Core)
       uid: migration/logging-nonaspnetcore
 - name: API reference
-  href: /dotnet/api/?view=aspnetcore-2.2
+  href: /dotnet/api/
```

Comment:
This URL will take the reader to the generic .NET API browser page: https://docs.microsoft.com/dotnet/api/. Is that the desired behavior? As far as I can tell, there's no way to link to an ASP.NET Core API ref landing page.

As an alternative, what if **API reference** were a top-level ToC node? Under it, we could include a link to each specific version's API ref landing page. It might look something like the following:

- API reference
  - ASP.NET Core 3.0 Preview 2
  - ASP.NET Core 2.2
  - ASP.NET Core 2.1
  - ASP.NET Core 2.0
  - ASP.NET Core 1.1
  - ASP.NET Core 1.0

Reply:
I forgot that. Let's just manually update to the latest released version.